.github/workflows/linting.yml

```
@@ -16,6 +16,8 @@ jobs:
         uses: actions/checkout@master
       - name: Run validation
         run: make codegen
+      - name: Make sure there's no pending changes
+        run: output=$(git status --porcelain) && [ -z "$output" ]
```

```suggestion
        run: git diff --exit-code
```

This will show the differences in the logs too.

---

Problem is that `git diff` doesnt count untracked files, so if such appear we'll miss them, while `git status` does

---

Good point, so

```suggestion
        run: git add -A && git diff --staged --exit-code
```